ntbs-service/Models/Notification.cs

```diff
@@ -81,7 +81,7 @@ public Notification()
         public string FullName => string.Join(", ", new[] { PatientDetails.FamilyName?.ToUpper(), PatientDetails.GivenName }.Where(s => !String.IsNullOrEmpty(s)));
         public string SexLabel => PatientDetails.Sex?.Label;
         public string EthnicityLabel => PatientDetails.Ethnicity?.Label;
-        public string CountryName => PatientDetails.Country?.Name;
+        public string CountryName => PatientDetails.Country?.Name ?? PatientDetails.LegacyCountryName?.ToLower();
```

Comment:
Not sure about this, migration database just holds a single string for birth country in all caps and need to grab it for the banner

Reply:
Hmm, if it's banner-specific logic, would it make more sense in the banner model?
But on the whole it seems right, it doesn't seem worth it trying to match it to a country we have in the db until we migrate it